draft-ietf-tls-wkech.xml

```diff
@@ -482,16 +487,59 @@ Client <-->|                    | |                    | facing
 
 
     <section title="Security Considerations">
+
+        <t>
+            This document defines a way to publish SVCB/HTTPS RR values.
+            If the wrong values were published in the DNS, then TLS clients
+            using ECH might suffer a privacy leak, or degraded service due to
+            overuse of ECH retry_configs.
+        </t>
+
+        <t>
+            When considering the content of SVCB/HTTPS RRs, the general argument for
+            the security of this scheme is that, assuming proper setup and
+            operation of the webPKI, this scheme has the backend server authenticate
+            the JSON structure that is mapped directly to the SVCB/HTTPS RR, to
+            eventually be used by TLS clients when interacting with the backend
+            server, via the client-facing server.
+        </t>
+
+        <t>
+            ECH split-mode security also requires that the backend server
+            acquire SvcParamKey values from the client-facing server via some
+            authenticated means. If the backend server acquires the JSON
+            data authenticated via HTTPS from the client-facing server's public_name
+            then that satisfies this requirement.
```

Comment:
I think this needs to specify "https://$public_name/.well-known/origin-svcb".  Fetching it from "https://$public_name/users/~somebody/config.json" might not be suitable.

Reply:
Changed to "If the backend server acquires the JSON data from the well-known URL and it is properly authenticated via HTTPS "